index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <a href="#">My logo</a>
+    </div>
+    <nav class="navlinks">
+      <img src="./Img/Hlogo.png">
+    </nav>
+  </header>
+  <section class="headline-main">
+    <div class="headline">
+      <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1>
+      <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+        or website
+        Look
+        through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <hx class="headline-sm-suport">LET’S CONNECT</hx>
```

Comment:
- [optional] nice job making the change. Unfortunately you didn't understand the previous reviewer well, when he said hx tag he meant since that is a heading you can use h2, h3, h4 etc and not `hx ` its self. The `x `there is for you to decide which one to use . Please fix that. You can learn more [here](https://www.w3schools.com/tags/tag_hn.asp).